src/documents/firstDiscoveryEditor.md

```diff
@@ -0,0 +1,385 @@
+
+<table>
+    <tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>
+    <tr>
+        <td>`defaultLocale`</td>
+        <td>The language code to use, for selecting the correct message bundle, when no language has been provided in the preferences.</td>
+        <td>A valid language code such as: `"en-US"`, `"fr-FR"`, `"es-MX"`, `"de-DE"`, `"nl-NL"`, `"sv-SE"`</td>
+        <td>
+        This resolves to "en-US":<br/>
+        <pre><code>defaultLocale: {
+    expander: {
+        funcName: "fluid.get",
+        args: [{
+            expander: {
+                funcName: "fluid.defaults",
+                args: ["gpii.firstDiscovery.schemas.language"]
+            }
+        }, ["schema", "properties", "gpii.firstDiscovery.language", "default"]]
+    }
+}</code></pre>
+        </td>
+    </tr>
+    <tr>
+        <td>`tooltipOptions`</td>
+        <td>The options to be used by any descendant tooltip components.</td>
+        <td></td>
+        <td>
+        <pre><codetooltipOptions: {
+    delay: 0,
+    duration: 0,
+    position: {
+        my: "left bottom",
+        at: "right+1 top"
+    },
+    styles: {
+        tooltip: "gpii-fd-tooltip"
+    },
+    items: ".gpiic-fd-tooltip:not([disabled])"
+}</code></pre>
+        </td>
+    </tr>
+    <tr>
+        <td>`selectors`</td>
+        <td>Javascript object containing selectors for various fragments of the markup, including the containers for the subcomponents.</td>
+        <td></td>
+        <td>See [Selectors](#selectors)</td>
+    </tr>
+    <tr>
+        <td>`styles`</td>
+        <td>Specific class names used to achieve the look and feel.</td>
+        <td></td>
+        <td>
+        This resolves to "en-US":<br/>
+        <pre><code>styles: {
+    active: "gpii-fd-active",
+    show: "gpii-fd-show",
+    currentPanel: "gpii-fd-current"
+}</code></pre>
+        </td>
+    </tr>
+    <tr>
+        <td>`modelListeners`</td>
+        <td>JavaScript object containing model paths and the listeners that are activated when changes happen to those paths.</td>
+        <td>Keys in the object are event names, values are functions or arrays of functions.</td>
+        <td>See [Model](#model)</td>
+    </tr>
+</table>
+
+## Selectors
+
+One of the options that can be provided to Infusion components is a set of CSS-based
+selectors identifying where in the DOM different elements can be found. Components use a
+[DOM Binder](http://docs.fluidproject.org/infusion/development/DOMBinder.html) to access the
+named elements.
+
+The value for the option is itself a Javascript object containing name/value pairs:
+
+```javascript
+selectors: {
+    selector1Name: "selector 1 string",
+    selector2Name: "selector 2 string",
+      ...
+}
+```
+
+| Selector Name | Description | Default |
+|---------------|-------------|---------|
+| `prefsEditor` | The container to use for the internal prefsEditor; which contains all of the adjuster panels. | `".gpiic-fd-prefsEditor"` |
```

Comment:
In the sentence "The container to use for the internal prefsEditor; which contains all of the adjuster panels.", should comma instead of semi-colon be used?